_episodes/11-joins.md

```diff
@@ -29,6 +29,10 @@ In this episode we will consider different scenarios and show we might join the
 
 ### Scenario 1 - Two data sets containing the same columns but different rows of data
 
+First, let's download the datafiles. You can download the ![pandas_join_types](../fig/pandas_join_types.png)
```

Comment:
Is it relevant and helpful to download this pandas join figure at this point for downloading data files?
It has already been displayed below.